careers/index.md

```diff
@@ -4,39 +4,38 @@ title:  Careers
+At Subnero, we believe that quality is key since our reputation drives our success. Quality comes naturally when everyone owns the task they undertake, does it with pride, and gives it their best. We also practice a culture of openness and continual improvement, giving and accepting timely feedback. We don't believe in being constrained by conventional thinking -- no idea is too crazy to discuss! We like people who think out of the box, innovate, and come up with solutions that push the boundaries of what is possible.
 
 ---
-## Electronics Engineer
+## Sales Executive
 
-As an Electronics Engineer, you will be responsible for architecting, designing, and conducting R&D testing of various electronics boards that form the building blocks of Subnero's products and solutions. Your expertise will contribute to the development and deployment of advanced underwater wireless networks.
+As a Sales Executive at Subnero, your role is pivotal in nurturing client relationships, driving business growth, and ensuring seamless sales processes. You'll be instrumental in expanding our market presence and fostering long-term partnerships.
 
 ### Roles & Responsibilities
-- Design, prototype, test, and debug digital and analog electronic systems.
-- Develop detailed schematics, layouts, and PCB designs using industry-standard design software (e.g. Altium software).
-- Evaluate and select appropriate components, ensuring their compatibility and availability for production.
-- Perform testing, simulation, and analysis of designs to ensure performance, reliability, and adherence to defined electrical specifications.
-- Liaising with various vendors for fabrication, assembly, and component sourcing.
-- Working with 3rd party test organizations for qualification testing (EMI/EMC).
-- Keep updated on industry trends, emerging technologies, and advancements in electronics design to drive innovation within the R&D projects.
+- Cultivate and expand existing client relationships, fostering growth and identifying new business opportunities.
+- Prepare quotations, engage in RFQs, and contribute to tender bids as needed, ensuring accuracy and timeliness.
+- Steer ongoing leads through the sales pipeline, driving them towards successful closures.
+- Coordinate customer activities, including meetings, follow-up sessions, and product demos, ensuring a smooth experience.
+- Implement post-sales follow-ups to secure repeat orders and maintain client satisfaction.
+- Manage and upkeep the CRM system for efficient tracking and organization of client interactions.
```

Comment:
Should we mention which CRM?

Reply:
I was reluctant to do this as:
1. The basic idea of CRM systems should be the same regardless of which system
2. We will provide training for our system anyway
3. They should be willing to learn, in case we switch in the future